### ISSUE TITLE
[Help Wanted] Unable to publish from windows machine to linux target

### ISSUE THREAD

**Issue by behzadk** [MonoGame/MonoGame#8500]
### Discussed in https://github.com/MonoGame/MonoGame/discussions/8498

<div type='discussions-op-text'>

<sup>Originally posted by **behzadk** September 23, 2024</sup>
Hi all, having issues running my app on a linux machine. The app is called `LifeGridMP`. I created it using the monogame template for multi platform using OpenGL using .NET6.0 framework.

I run this on my windows machine and move `./artifacts/linux-x64` to my linux machine - Raspberry pi 4, Model B running NOOBS os.
```
dotnet publish ./LifeGridMP.sln --configuration Release --output ./artifacts/linux-x64--self-contained true --runtime linux-x64
```

On my linux machine I have mono installed.
```
mono LifeGridMP.dll
```

And receive this error (I have masked the paths with `<PATH>`)
```wrap
Unhandled Exception:
System.TypeLoadException: Could not set up parent class, due to: Could not set up parent class, due to:  assembly:<PATH>System.Private.CoreLib.dll type:Object member:(null) assembly:<PATH>/MonoGame.Framework.dll type:Game member:(null)
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not set up parent class, due to: Could not set up parent class, due to:  assembly:<PATH>System.Private.CoreLib.dll type:Object member:(null) assembly<PATH>MonoGame.Framework.dll type:Game member:(null)
```

Not really sure where to start debugging this. All those .dll files exist. Any advice on how to address this?</div>

**Comment by behzadk** [comment]
Properties
![image](https://github.com/user-attachments/assets/9d55f470-5854-4572-b411-95fa7b0b3541)

I have also tried publishing with the UI and get the same error when I try and run it (tried with linux-arm64 and linux-x64)
![image](https://github.com/user-attachments/assets/a0d1f3c6-035c-4e05-ad65-08fc351ad806)

**Comment by behzadk** [comment]
Issue was solved in the discussion! #8498

For reference: 

In the end In had two problems:

1. I was building for `linux-x64`. I'm running on a Raspberry Pi, so I needed the `linux-arm` release
2. Mono is not needed and I can execute from command-line after changing permissions
```
chmod +x ./LifeGridMP
./LifeGridMP
```
Or by double clicking the file using the GUI
